ert3/stats/_stats.py

```diff
@@ -2,85 +2,67 @@
 import scipy.stats
 
 
-class Gaussian:
-    def __init__(self, mean, std, *, size=None, index=None):
-        self._mean = mean
-        self._std = std
-
+class Distribution:
+    def __init__(self, *, size, index):
         if size is None and index is None:
             raise ValueError(
-                "Cannot create gaussian distribution with neither size nor index"
+                "Cannot create distribution with neither size nor index"
             )
         if size is not None and index is not None:
             raise ValueError(
-                "Cannot create gaussian distribution with both size and index"
+                "Cannot create distribution with both size and index"
             )
 
-        self._index = index
         if size is not None:
             self._size = size
+            self._as_array = True
+            self._index = tuple(range(self._size))
         else:
             self._size = len(tuple(index))
-            self._index = tuple(index)
+            self._as_array = False
+            self._index = index
```

Comment:
Is it intentional to not explicitly convert index to tuple as before? If is, then the len statement on line 21 should probably also change.

Reply:
Nope. I don't think so at least. Had to do some changes later due to `Mypy`, but uncertain whether this is one of them... So this change should rather occur then, if necessary...